Clarify some loading tips

Rhundas opened the pull request:
Some of the current loading tips are potentially misleading, or could be clarified further.

Rhundas committed:
Clarify some loading tips

Some of the current loading tips are potentially misleading, or could be clarified further.

Janaszar commented on RogueTech Core/GameTips/general.txt:
```diff
@@ -1,23 +1,23 @@
-Shift-Click on the skills tab of a pilot in the Barracks to reset their skills; Requires Argo Training Module Upgrade.
+CTRL+Click lets you jettison ammo bins to avoid heat-related explosions!
+CTRL+A toggles visibility of ECM bubbles.
+If you have the Argo Training Module Upgradem you can Shift+Click on the skills tab of a pilot in the Barracks to reset their skills.
```
This requires the second training module upgrade, right?

```suggestion
If you have the second Argo Training Module Upgrade you can Shift+Click on the skills tab of a pilot in the Barracks to reset their skills.
```
At the very least, there's an `m` on Upgrade.

Rhundas replied:
i can type don't bully me :<

Janaszar commented on RogueTech Core/GameTips/general.txt:
```diff
@@ -26,39 +26,38 @@ Positioning is key. Take the high ground or dig in deep.
 A 'Mech needs an engine core to function. 
 An XL engine will save a lot of weight, but if you lose a Torso Side, you lose the 'Mech.
-Bring some streaks or pulse lasers to deal with those pesky Lights!
+Weapons with accuracy bonuses or evasion ignoring, like Pulse Lasers, are fantastic against fast-moving enemies.
```
```suggestion
Weapons with accuracy bonuses or that can ignore evasion, such as Pulse Lasers, are fantastic against fast-moving enemies.
```

Janaszar commented on RogueTech Core/GameTips/general.txt:
```diff
@@ -26,39 +26,38 @@ Positioning is key. Take the high ground or dig in deep.
+Weapons with accuracy bonuses or evasion ignoring, like Pulse Lasers, are fantastic against fast-moving enemies.
 Watch your heat. Riding the redline will slow you down or blow you sky high.
-Riding the redline is dangerous... but fun.
+Riding the redline is dangerous...but fun.
```
```suggestion
Riding the redline is dangerous... but fun.
```
As far as I know, an ellipse with following text gets a space between the two.

Rhundas replied:
This depends entirely on the style guide employed for writing. I've reached out to HBS to see if their writers can comment on what style guide was used for HBS BattleTech. When I get a response, I'll adopt whatever it is that they're using.

In the meantime, I'll leave this as [UK (ONS) style](https://style.ons.gov.uk/house-style/ellipsis/) for internal consistency. See commit https://github.com/BattletechModders/RogueTech/pull/4298/commits/d87638adbfe4c438ad60df4330b00cbaa50427fe

Janaszar commented on RogueTech Core/GameTips/general.txt:
```diff
@@ -26,39 +26,38 @@ Positioning is key. Take the high ground or dig in deep.
 IFF jammers add a penalty to enemy sensor check's against a unit, making it harder to detect.
-UAV's allow you to do a quick scan of the Battlefield and Sensor Lock every Nearby Enemy.
-Melee combat in Roguetech is dramatically more difficult to start with, but a specialized melee 'Mech is a nightmare.
+UAVs allow you to do a quick scan of the battlefield and sensor lock every nearby enemy.
```
```suggestion
UAV's allow you to do a quick scan of the battlefield and sensor lock every nearby enemy.
```
Plurals of abbreviations/acronyms tend to use `'s` in my experience.  It's not wrong as-is, just uncommon/rare to me.

Janaszar commented on RogueTech Core/GameTips/general.txt:
```diff
@@ -26,39 +26,38 @@ Positioning is key. Take the high ground or dig in deep.
+Melee combat in RogueTech is dramatically more difficult to start with, but a specialized melee 'Mech is a nightmare.
 Railguns and Gauss rifles deal tremendous damage at equally tremendous ranges. Reach out and touch someone.
-Killing Rocket Launcher or RL units before they blow their load is tactically advisable.
+Use your sensors to check enemy loadouts! 'Mechs and vehicles with rocket launchers (RLs) can do a lot of damage.
```
Feels like the old one is fine, and this should be a new one?
Maybe
`Better sensor rolls allow you to to check enemy loadouts!  You'll want to know if that truck is carrying a fluid gun, or something more sinister.`

Rhundas replied:
I felt that the innuendo implied was a little stale, personally, but obviously it's not my work so if people want it kept then that's fine.

Rhundas replied:
Reverted in commit https://github.com/BattletechModders/RogueTech/pull/4298/commits/3ccb47eec110b546d94250908361e9a526950a4e

Janaszar commented on RogueTech Core/GameTips/general.txt:
```diff
@@ -26,39 +26,38 @@ Positioning is key. Take the high ground or dig in deep.
+Use your sensors to check enemy loadouts! 'Mechs and vehicles with rocket launchers (RLs) can do a lot of damage.
 Melee attacks against the side and rear of a target have an increased chance to hit.
-Overheating 'Mechs in Roguetech do not suffer structure damage, but their ammunition bins have a small chance to explode.
+Overheating 'Mechs in Roguetech do not suffer structure damage, but they take penalties to speed and aim. If you run hot enough, your ammo can explode, too!
```
```suggestion
Overheating 'Mechs in Roguetech do not suffer structure damage, but they take penalties to speed and to-hit. If you run hot enough, you can shutdown, take an injury, and your ammo can explode too!
```
Aim just sounds weird?
Added a bit on what else can happen with overheating.

Janaszar commented on RogueTech Core/GameTips/general.txt:
```diff
@@ -67,8 +66,8 @@ After a forest tile has been burned, it offers no cover and negatively affects t
 90% to hit is still 10% to miss. Dice don't follow your expectations.
 Friendly Fire isn't friendly. Don't put your mechs in the line of fire.
 Pyrrhic victories are for idealists, you're a merc. If the repairs are going to cost more than you are getting paid you should have left already.
```
```suggestion
Pyrrhic victories are for idealists; you're a merc. If the repairs are going to cost more than you are getting paid you should have left already.
```
The two clauses are of opposing intents.

Janaszar reviewed (commented):
Was linked to this from the Discord, where it was requested that a native English speaker review it.  (I'm pretty sure I fit that, though I know I have some quirks.)

Went through and added comments/possible changes as I saw fit.

Rhundas committed:
Update RogueTech Core/GameTips/general.txt

Co-authored-by: Janaszar <32232509+Janaszar@users.noreply.github.com>

Rhundas committed:
Update RogueTech Core/GameTips/general.txt

Co-authored-by: Janaszar <32232509+Janaszar@users.noreply.github.com>

Rhundas committed:
Update RogueTech Core/GameTips/general.txt

Co-authored-by: Janaszar <32232509+Janaszar@users.noreply.github.com>

Rhundas committed:
Update RogueTech Core/GameTips/general.txt

Co-authored-by: Janaszar <32232509+Janaszar@users.noreply.github.com>

Rhundas committed:
Update RogueTech Core/GameTips/general.txt

Co-authored-by: Janaszar <32232509+Janaszar@users.noreply.github.com>

Rhundas committed:
Fix grammatical error with ellipsis.

According to UK style guides, ellipsis mid-sentence have spaces on either side.
https://style.ons.gov.uk/house-style/ellipsis/

Rhundas committed:
Pass 2. Fix spacing and grammar. Add additional tips. Add additional 'tips'.

Rhundas committed:
Don't aren't.

:)

Rhundas committed:
Stop using the word scrap so many times.

Rhundas committed:
Pass 3. Revert change to RL units. Add tip re: sensor rolls.

Rhundas committed:
Small vocabulary change to be internally consistent

Changed "mouseover" to "hovering your cursor over" in line 12 to match line 6.

Rhundas committed:
Clarify pilot retrain tip.

Added the full name of the upgrade to avoid any confusion.

Rhundas committed:
Fix remaining ellipsis spacing. 

Missed a few in the last pass.
